e2e/test/scenarios/sharing/reproductions.cy.spec.js

```diff
@@ -259,49 +259,6 @@ function setUpAlert(questionId) {
   });
 }
 
-describe("issue 16918", () => {
```

Comment:
i wonder if we need to replace this with a reproduction for dashboard subscriptions

Reply:
Absolutely yes!

Reply:
om, so could you help me with that? :D

Reply:
https://github.com/metabase/metabase/pull/35843/commits/43eaf41b45a5fcc683d0d2967e3f15b7fc913592

Reply:
thank you thank you!